config/manifests/kustomization.yaml

```diff
@@ -25,5 +25,3 @@ vars:
     fieldpath: data.odh-codeflare-operator-controller-image
 resources:
 - ../default
-- bases/codeflare-notebook-imagestream.yaml
-
```

Comment:
Could be removed to make pre-commit happy.

Reply:
Weird... The file looks correct to me locally.

<img width="362" alt="image" src="https://github.com/red-hat-data-services/codeflare-operator/assets/6526150/0d1f61b8-3976-4317-a9b0-73d1955f9194">